### PR TITLE
Home work 5

### PR DESCRIPTION
**Pull request by Davydovychyurii**

### CODE REVIEW

File: HW5/src/js/task2.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                };
+                break;
+            };
+            let userNumber = +prompt(`Game Level: ${levelGame} 
+            Enter a number range from 0 to ${magicalNumber}
+            Remained attempts: ${attempts}
+            Total prize: ${totalPrize}$
+            Possible prize on current attempts: ${Math.trunc(jackpot / i)}$`);
+            attempts--;
+            console.log(correctNumber);
```
Comment (AgentPrus) [inline HW5/src/js/task2.js]: remove console.log

File: HW5/src/js/task2.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    let jackpot = 10;
+    let attempts = 3;
+    let levelGame = 1;
+    while (gameStatus) {
+        let correctNumber = Math.round(Math.random() * magicalNumber);
+        for (let i = 1; i <= 5; i++) {
+            if (i == 3) {
+                continue;
+            };
+            console.log(correctNumber);
```
Comment (AgentPrus) [inline HW5/src/js/task2.js]: remove console.log

File: HW5/src/js/task2.js (hunk 1 above)
Comment (Davydovychyurii) [reply HW5/src/js/task2.js]: +

File: HW5/src/js/task2.js (hunk 2 above)
Comment (Davydovychyurii) [reply HW5/src/js/task2.js]: +

**Review by Kiril99 — APPROVED** [review]

**Pull request merged**